endpoints/openrtb2/auction.go

```diff
@@ -133,8 +123,15 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 	defer cancel()
 
 	usersyncs := usersync.ParsePBSCookieFromRequest(r, &(deps.cfg.HostCookie))
+	if req.Site != nil && req.Site.Publisher != nil {
+		labels.PubID = req.Site.Publisher.ID
+	}
 	if req.App != nil {
```

Comment:
I am assuming that the request will be either from an `app` or a `site` so wouldn't it be good to have a sanity check that will return an error if both `req.Site` and `req.App` are not nil for some reason unless that is already done at a higher level in the code?

Reply:
Fair point. Changes made

Reply:
I am sorry Gus but I don't see that check in the update. Can you point me to it?

Reply:
Ohh the error check you mean? but if there's no `req.Site` nor `req.App` we just set to "UNKNOWN" isn't it? Do you mean that we should implement an additional error message?

Reply:
I meant that we check if both `req` and `app` are set and if they are then return an error.